models/notification.py

```diff
@@ -16,7 +16,7 @@ class Notification(object):
     """
 
     @staticmethod
-    def send(template_type: str, to_email: str, content: dict):
+    def send(template_type: str, to_email: str, content: dict, reference: str | None = None):
```

Comment:
Shall we just call it `govuk_notify_reference` all the way so the consuming code reads the same? I guess this would let it be more generic but I suspect this interface wont need to think too far into the future

Reply:
Sure 👍